exchanges/huobi/huobi_test.go

```diff
@@ -1825,30 +1827,27 @@ func TestPairFromContractExpiryCode(t *testing.T) {
 	today := time.Now()
 	today = time.Date(today.Year(), today.Month(), today.Day(), 0, 0, 0, 0, tz) // Do not use Truncate; https://github.com/golang/go/issues/55921
 
-	for _, cType := range contractExpiryNames {
-		p, err := e.pairFromContractExpiryCode(currency.Pair{
-			Base:  currency.BTC,
-			Quote: currency.NewCode(cType),
+	require.NotEmpty(t, e.futureContractCodes, "At least one contract code must be loaded")
+
+	for cType, cachedContract := range e.futureContractCodes {
+		t.Run(cType, func(t *testing.T) {
+			t.Parallel()
+			p, err := e.pairFromContractExpiryCode(currency.Pair{
+				Base:  currency.BTC,
+				Quote: currency.NewCode(cType),
+			})
+			require.NoError(t, err)
+			assert.Equal(t, currency.BTC, p.Base, "pair Base should be BTC")
+			assert.Equal(t, cachedContract, p.Quote, "pair Quote should match contractExpiryNames")
```

Comment:
contractExpiryNames is no longer referenced

Reply:
Fixed https://github.com/gbjk/gocryptotrader/commit/8dc1d9c44